apiserver/facades/client/bundle/bundle.go

```diff
@@ -151,10 +156,111 @@ func (b *BundleAPI) GetChanges(args params.BundleChangesParams) (params.BundleCh
 }
 
 // ExportBundle exports the current model configuration as bundle.
-func (b *BundleAPI) ExportBundle() (params.StringResult, error) {
-	return params.StringResult{}, nil
+func (b *BundleAPI) ExportBundle() (params.BytesResult, error) {
+	if err := b.checkCanRead(); err != nil {
+		return params.BytesResult{}, common.ServerError(common.ErrPerm)
+	}
+
+	var exportConfig state.ExportConfig
```

Comment:
To simplify the extraction of the model information, we are taking advantage of the ExportPartial method on state. Since this needs the ExportConfig, we are including state here for expediency.

Perhaps in the future we may add an ExportForBundle method on state if we really want to not import state here, but until we know more we aren't doing that.